app/templates/agendapoints/edit.hbs

```diff
@@ -1,5 +1,10 @@
-<AppChrome @documentType="draftDecisions" @editorDocument={{this.editorDocument}} @documentContainer={{this.documentContainer}} @save={{action "save"}} @sendToTrash={{action "sendToTrash"}} @exportHtmlFunction={{action "download"}} />
-{{!-- Hide toolbar until implemented {{#editor-toolbar}}{{/editor-toolbar}} --}}
+<AppChrome
+  @editorDocument={{this.editorDocument}}
+  @documentContainer={{this.documentContainer}}
+  @save={{hash action=this.save isRunning=this.saveTask.isRunning}}
```

Comment:
you could use the perform helper here, which would mean you can get rid of the this.save method in the js file
`hash action=(perform this.saveTask) isRunning=this.saveTask.isRunning`